js/views/userPage/Home.js

```diff
@@ -12,11 +12,8 @@ export default class extends BaseVw {
   constructor(options = {}) {
     super(options);
     this.options = options;
-
     this.ownPage = options.ownPage;
-
-    this.userCard = new UserCard({ model: this.model, hideControls: true });
-
+    this.userCard = this.createChild(UserCard, { model: this.model, hideControls: true });
```

Comment:
@jjeffryes Do you know why we are hiding the controls on the user card on the home page?

I understand to not show them if it's your own page, but I'm not sure why we'd otherwise be hiding the Moderator, Blocked and Follow toggles.

Reply:
I don't think the design had them on the home page.

@morebrownies should that be revised?

Reply:
@morebrownies any thoughts on this?

Reply:
Yeah we can show them on the home page too